apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/Defendant.tsx

```diff
@@ -117,6 +117,8 @@ const Defendant = () => {
   const { updateIndictmentCount, deleteIndictmentCount } = useIndictmentCounts()
 
   const [policeCases, setPoliceCases] = useState<PoliceCase[]>([])
+  const [isProsecutorSelected, setIsProsecutorSelected] =
+    useState<boolean>(false)
```

Comment:
Shouldn't this be initialized with the existing prosecutor on the case?

Reply:
Discussed IRL, this is only for this step's validation, i.e. we only need to know if a prosecutor has been selected or not.